.github/workflows/dummy-server-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test dummy-server
```

Comment:
I'm not a huge fan of testing a _testing utility_ - seems redundant. If dummy-server is broken, the tests it's being used in will be broken. And those tests are (currently) not testing anything relevant anyway...

But I'm OK with testing it. What I definitely don't like is _it having its own workflow_. We should just use `./build.js test` for that. If we have a strong urge to test _dev-utils_ separately from everything else (I don't see why, but say, "if"), we should just add that logic to `build.js` IMHO.

Every bit of logic we have in GitHub Action workflow files is us losing against complexity IMHO.

Reply:
In that case, you will be doing implicit rather than explicit testing, and you would need to add all kind of path detection to only do what's strictly needed for this. In general, whatever you do in that sense will be more complicated. This follows the F.I.R.S.T principles, it fails early and avoids triggering the whole nine yards if only this is changed.
I would be for doing a common test for *all* dev-utils (right now, the other one fell off the testing bin too). But being as they are *test* dependencies, adding another state to the testing stack would *really* make it more complicated.